### DISCUSSION TITLE
Problems testing modules that use vue config helpers

### DISCUSSION THREAD

**Discussion by groteck** [nuxt/nuxt#18113] [Questions] [answered]
I'm running into an issue that testing a module that use the vue config helper, the tests are not able to resolve the config import, for example:
```ts
// Module that will be tested
import config from "#config";
// Code that will be tested
```

```ts
// Test file for the module
import {codeToBeTested} from "./module-to-be-tested";
// test("", () => {
// }
```

The error is:
```
Cannot find module '#config' from 'services/module-to-be-tested'
```

I'm using Jest and suming that I will need to import and initialize some test helpers.

**Comment by tobiasdiez** [comment, marked as answer]
Adding `#config` to jest's moduleNameWrapper worked for me:
https://github.com/JabRef/JabRefOnline/pull/480/files#diff-860bd1f15d1e0bafcfc6f62560524f588e6d6bf56d4ab1b0f6f8146461558730R9
